rt_dashboard/javascript/src/components/dashboard/dashboard.js

```diff
@@ -0,0 +1,29 @@
+import html from './dashboard.html'
+
+class Dashboard extends HTMLElement {
+  constructor () {
+    super()
+    const template = document.createElement('template')
+    template.innerHTML = html
+
+    this.attachShadow({ mode: 'open' })
+    this.shadowRoot.appendChild(template.content.cloneNode(true))
+
+    this.selectedQueueChange = this.selectedQueueChange.bind(this)
+  }
+
+  connectedCallback () {
+    const queuesComponent = this.shadowRoot.querySelector('queues-component')
+    queuesComponent.addEventListener('selectedQueueChange', this.selectedQueueChange)
+  }
+
+  disconnectedCallback () {
+    this.removeEventListener('selectedQueueChange', this.selectedQueueChange)
+  }
+
+  selectedQueueChange (event) {
+    const { detail: { queueName } } = event
+    console.log(queueName)
+  }
+}
+window.customElements.define('dashboard-component', Dashboard)
```

Comment:
I suggest to extract the registration into a `index.js`, that way if for whatever reason we don't need it registered, we can use the class directly.

Also I suggest to use `rt-` prefix.